templates/common/_base/files/tuned-default-conf.yaml

```diff
@@ -0,0 +1,13 @@
+filesystem: "root"
+mode: 0644
+path: "/etc/tuned.conf"
+contents:
+  inline: |
+    [main]
+    summary=Optimize systems running OpenShift nodes
+    include=openshift
+
+    [sysctl]
```

Comment:
do we want to use the sysctl plugin of tuned here, or configure sysctl directly?